include/openssl/hpke.h

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright 2019-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/**
+ * @file
+ * Data structures/prototypes for HPKE (RFC9180)
+ */
+
+#ifndef HPKE_H_INCLUDED
+# define HPKE_H_INCLUDED
+
+# include <openssl/ssl.h>
+
+/** biggest/default buffer for keys and internal buffers we use */
+# ifndef HPKE_MAXSIZE
+#  define HPKE_MAXSIZE (2 * 1024) /* 2k: enough for anyone :-) */
```

Comment:
All other macros here should be prefixed with `OSSL_`

Reply:
yep did that ta